packages/docs/docs/docs/migrate_from_1.x_to_2.x.md

```diff
@@ -0,0 +1,117 @@
+---
+id: migrate_from_1.x_to_2.x
+title: Migrating to 2.x
```

Comment:
Maybe we can call it: What's new in ZeppelinOS 2.0?

Reply:
+1! Besides, we'll eventually write an actual migration guide, as we script the process